packages/network-controller/tests/fake-provider-engine.ts

```diff
@@ -117,9 +112,20 @@ export class FakeProviderEngine extends EventEmitter implements ProviderEngine {
     this.#isStopped = true;
   }
 
-  sendAsync<P, V>(
-    payload: SendAsyncPayload<P>,
-    callback: (error: unknown, response: RpcResponse<RpcPayload<P>, V>) => void,
+  send = <P>(payload: JsonRpcRequest<P>) => {
```

Comment:
Nit: Is there a reason to use `=`? Do we need to bind this? Otherwise we could stick with `send<P>` and `sendAsync<P>` which would be consistent with the other methods here.

Reply:
I get this error:
```
 tests/fake-provider-engine.ts:115:3 - error TS2425: Class 'SafeEventEmitterProvider' defines instance member property 'send', but extended class 'FakeProvider' defines it as instance member function.

    115   send<P>(payload: JsonRpcRequest<P>) {
```